docs/pages/distribution/publishing-websites.mdx

```diff
@@ -26,153 +43,246 @@ If you make changes to your project, you'll need to rebuild it for production. D
 
+</Tab>
+
+</Tabs>
+
 ## Hosting on third-party services
 
-### AWS Amplify Console
+### Netlify
 
-The [AWS Amplify Console](https://console.amplify.aws) provides a Git-based workflow for continuously deploying and hosting full-stack serverless web apps. Amplify deploys your PWA from a repository instead of from your computer. In this guide, we'll use a GitHub repository. Before starting, [create a new repo on GitHub](https://github.com/new).
+[Netlify](https://www.netlify.com/) is a mostly-unopinionated platform for deploying web apps. This has the highest compatibility with Expo web apps as it makes few assumptions about the framework.
+
+#### Manual deployment with the Netlify CDN
 
 <Step label="1">
 
-Add the [**amplify-explicit.yml**](https://github.com/expo/amplify-demo/blob/master/amplify-explicit.yml) file to the root of your repo.
+Install the Netlify CLI by running the following command:
+
+<Terminal cmd={['$ npm install -g netlify-cli']} />
 
 </Step>
 
 <Step label="2">
 
-Push your local Expo project to a GitHub repository. If you haven't pushed to GitHub yet, follow [GitHub's guide to add an existing project to GitHub](https://docs.github.com/en/get-started/importing-your-projects-to-github/importing-source-code-to-github/adding-locally-hosted-code-to-github).
+Configure redirects for single-page applications.
 
-</Step>
+<Tabs>
 
-<Step label="3">
+<Tab label="Expo Router">
 
-Login to the [Amplify Console](https://console.aws.amazon.com/amplify/home) and choose **Get started** under **Deploy**. Grant Amplify permission to read from your GitHub account or the organization that owns your repo.
+> If your app uses [static rendering](/router/reference/static-rendering), then you can skip this step.
 
-</Step>
+`expo.web.output: 'single'` generates a single-page application. This means there's only one **dist/index.html** file which all requests must be redirected to. This can be done in Netlify by creating a `./public/_redirects` file and redirecting all requests to `/index.html`.
 
-<Step label="4">
+```sh public/_redirects
+/*    /index.html   200
+```
 
-The Amplify Console will detect that the **amplify.yml** file is in your repo. Choose **Next**.
+If you modify this file, you will need to rebuild your project with `npx expo export -p web` to have it safely copied into the **dist** directory.
 
-</Step>
+</Tab>
 
-<Step label="5">
+<Tab label="webpack">
 
-Review your settings and choose **Save and deploy**. Your app will now be deployed to a `https://branchname.xxxxxx.amplifyapp.com` URL.
-
-</Step>
+If your app implements any navigation, you'll need to configure Netlify to redirect requests to the single **web-build/index.html** file. This can be done in Netlify by creating a `./public/_redirects` file and redirecting all requests to `/index.html`.
```

Comment:
```suggestion
If your app implements any navigation, you'll need to configure Netlify to redirect requests to the single **web-build/index.html** file. This can be done in Netlify by creating a **./public/_redirects** file and redirecting all requests to **/index.html**.
```